playground/frontend/pubspec.yaml

```diff
@@ -66,7 +68,6 @@ dev_dependencies:
   mockito: ^5.0.16
   build_runner: ^2.1.4
   google_fonts: ^2.3.1
-  usage: ^4.0.2
```

Comment:
This package had been listed under `dev_dependencies` while being used at runtime. This had been producing linter issues.